tile_geotiff.py

```diff
@@ -205,15 +209,163 @@ def move_imgs(directory: str) -> None:
     """ Moves all images within each sub directory into one directory """
     f_path = os.path.join(config.DATASETS_DIR, args.directory)
     for root, directories, files in os.walk(f_path, topdown=False):
-        for imgs in files:
+        for img in files:
             os.rename(
-                os.path.join(root, imgs),
-                os.path.join(f_path, imgs)
+                os.path.join(root, img),
+                os.path.join(f_path, img)
             )
         for name in directories:
             os.rmdir(os.path.join(root, name))
 
 
+def groom_imgs(directory: str) -> None:
+    VH_REGEX = re.compile(r"(.*)\.tile\.vh\.tif")
+    f_path = os.path.join(config.DATASETS_DIR, args.directory)
+    g_path = os.path.join(config.DATASETS_DIR, f'{args.directory}Groomed')
+
+    done = False
+    count = 0
+    update_count = 0
+    for root, directories, files in os.walk(f_path):
+        for vh in files:
+            if done:
+                break
+            m = re.match(VH_REGEX, vh)
+            if not m:
+                continue
+
+            pre = m.group(1)
+            mask = f"{pre}.mask.tif"
+            vv = f"{pre}.tile.vv.tif"
+
+            # Contains full path and the name for each image
+            l_imgs = [
```

Comment:
You could make this a list of tuples:
```python
l_imgs = [
    (mask, os.path.join(root, mask)),
    (vh, os.path.join(root, vh))
]
```
Or you could just do the joining in the places you need it and only store mask, vh, vv in this list.

*Note: This is not very important, and more of a hyper-optimization thing*

If you know the number of elements will be constant, it's often better to use a tuple instead of a list since they are more space efficient. 
```python
l_imgs = (
    item1,
    item2,
...
)
```